azblob/options.go

```diff
@@ -62,6 +63,12 @@ func WithListPrefix(prefix string) Option {
 	}
 }
 
+func WithListMaxResults(maxResults int32) Option {
```

Comment:
minor Function comment, I think this is pretty self explainatory, but it would be nice to know if its how much a single list request returns, i.e. page size

Reply:
added